content/english/ml-machine-learning/01-regression.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "What is regression?"
+description: "Make computers learn to predict outcomes."
+prereq: "Python"
+icon: ""
+draft: false
+weight: 1
+---
+
+# What is regression?
+Regression is a technique to model the relationship between a feature (independent variables) and a variable feature (dependent variables). It helps us understand how the value of a dependent variable is changing based on the value of the independent variable. When applied properly it helps us predict values 😯!
+
+|![Regression](./resources/regression.png)|
```

Comment:
I think that we should separate this even further into steps. First, show just the scatter plot relating the variables. Then explain that we see that the data looks kind of like a line and so we want to find a line that represents the data, then show a picture of a line that isn't representative, and explain why that's a bad line, then another picture of the actual regression line on top of the scatterplot as the culmination of this process explaining why this is a good fit. The regression line is our objective, so I think we've gotta work toward it instead of having it as a given.